src/Network.cpp

```diff
@@ -335,18 +344,15 @@ template <size_t spatial_size>
 void batchnorm(size_t channels,
                std::vector<float>& data,
                const float* means,
-               const float* variances,
+               const float* scale_stddivs,
```

Comment:
Name the parameter stddivs. (Which can then be made consistent with the global var name in Network) The scaling is done inside the function, it's not the stddiv that is scaled.